src/constants/_constants/eventStatsStatusFields.js

```diff
@@ -0,0 +1,63 @@
+import {
+  COLORS
+} from "./theme";
+
+export const APPLICATION_STATUS = {
+  "accepted": {
+    dbValue: "accepted",
+    label: "Accepted",
+    color: COLORS.LIGHT_GREEN,
+    getWindowMessage: (fname, lname) => `Do you want to accept ${fname} ${lname}?\nThis will send an email to the user.`
```

Comment:
waitlist/reviewing seems to be the only statusses that send an email to the user. Either way have all of them send email or just cahnge the window message to not say an email will be sent?

Reply:
Is the latter just to ensure that event directors aren't confused when there isn't a popup? If so, i think we can just verbally let them know that they have to send acceptances/rejections manually themselves.

Reply:
yeah the latter just so they dont get confused, I feel like we'll have people asking if the messages all say that this will send an email, maybe change it to, NOTE this will not send an email, it will have to be manually sent or smth

Reply:
I kind of disagree. I lowkey would hate if i had to manually click the popup every time i accepted someone. It just makes processes slower.

Reply:
right now the popup happens either way